### PR TITLE
AMB-1354: Separate Auth NHS Login

### PR DESCRIPTION
**Pull request by ChewieCB**

### CODE REVIEW

File: tests/test_endpoints.py [hunk 1]
```diff
@@ -68,25 +71,10 @@ def test_wait_for_status(nhsd_apim_proxy_url, status_endpoint_auth_headers):
     assert deployed_commitId == getenv('SOURCE_COMMIT_ID')
 
 
-@pytest.mark.nhsd_apim_authorization({"access": "application", "level": "level0"})
-def test_app_level0(nhsd_apim_proxy_url, nhsd_apim_auth_headers):
-    resp = requests.get(f"{nhsd_apim_proxy_url}", headers=nhsd_apim_auth_headers)
-    assert resp.status_code == 401  # unauthorized
```
Comment (RyanThomas1214) [inline tests/test_endpoints.py:74]: Maybe add this test_app_level0 test back in so we have an example of the VerifyAccessToken policy rejecting something as well?

Other than that all looks good

Comment (ChewieCB) [reply tests/test_endpoints.py:74]: f14c5f581fd633d7b5fd7a4661dd7fb3a65a541c

**Review by RyanThomas1214 — APPROVED** [review]

**Pull request merged**